plugin.video.digistorageplayer/addon.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.digistorageplayer" name="Digi Storage Player" version="1.0.0" provider-name="RCSRDS">
+	<requires>
+		<import addon="xbmc.python" version="2.14.0"/>
```

Comment:
If you want this available in Jarvis and above versions, "xbmc.python" should be set to "2.24.0" , your current setting is for Gotham. Please see: http://kodi.wiki/view/Addon.xml#Dependency_versions for more info.